xml/signals/FS-1987/appearance-ACEIavv.1vela.xml

```diff
@@ -122,13 +128,20 @@ Signal Rules “Regolamento Segnali” of the Italian State Railways (“Ferrovi
 
 
   </appearances>
+  <specificappearances>
 
+  </specificappearances>
```

Comment:
Headless CI fails on this since this tag is empty. It's not stored in the new file. So I suggest you remove this tag.
```
ERROR - match failed in LoadAndStoreTest: [main] jmrit.signalsystemeditor.configurexml.LoadAndStoreAllSignalSystemsTest.checkFile()
ERROR -     file1:line 122: "<aspectMappings>" [main] jmrit.signalsystemeditor.configurexml.LoadAndStoreAllSignalSystemsTest.checkFile()
ERROR -     file2:line 132: "<specificappearances>" [main] jmrit.signalsystemeditor.configurexml.LoadAndStoreAllSignalSystemsTest.checkFile()
ERROR -   comparing file1:"/tmp/junit1700853673038735376/xml/signals/FS-1987/appearance-ACEIavv.1vela.xml" [main] jmrit.signalsystemeditor.configurexml.LoadAndStoreAllSignalSystemsTest.checkFile()
ERROR -          to file2:"xml/signals/FS-1987/appearance-ACEIavv.1vela.xml" [main] jmrit.signalsystemeditor.configurexml.LoadAndStoreAllSignalSystemsTest.checkFile()
```